public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx

```diff
@@ -72,6 +76,11 @@ const EpicTestVariantEditor: React.FC<EpicTestVariantEditorProps> = ({
 }: EpicTestVariantEditorProps) => {
   const classes = getUseStyles(epicEditorConfig.allowMultipleVariants)();
 
+  const templateValidator =
+    epicEditorConfig.platform && epicEditorConfig.platform === 'AMP'
+      ? ampInvalidTemplateValidator
+      : invalidTemplateValidator;
```

Comment:
We can drop the initial `epicEditorConfig.platform &&` check now it's not optional, but do we need to check for `epicEditorConfig.platform === 'APPLE_NEWS'` here?

Reply:
✅